spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -547,28 +557,24 @@ public void testSparkTableReservedProperties() throws Exception {
     String[] keys = {"provider", "format", "current-snapshot-id", "location", "sort-order"};
 
     for (String entry : keys) {
-      Assert.assertTrue(
-          "Created table missing reserved property " + entry,
-          table.properties().containsKey(entry));
+      assertThat(table.properties())
+          .as("Created table missing reserved property " + entry)
+          .containsKey(entry);
```

Comment:
nit: We could use `containsKeys` instead:
```java
    assertThat(table.properties())
            .as("Created table missing reserved properties")
            .containsKeys(keys);
```

Also, `as` method accepts string format style: 
```java
          .as("Created table missing reserved property %s", entry)
```

Reply:
Thank you! Let me update the Spark 3.5 as well (and thanks for pointing the string style for `as`)